exporters/elasticsearch-exporter/src/main/java/io/zeebe/exporter/ElasticsearchClient.java

```diff
@@ -179,37 +181,42 @@ public boolean putIndexTemplate(
     template.put("index_patterns", Collections.singletonList(templateName + INDEX_DELIMITER + "*"));
 
     // update alias in template in case it was changed in configuration
-    template.put("aliases", Collections.singletonMap(aliasName, Collections.EMPTY_MAP));
-
-    final PutIndexTemplateRequest request =
-        new PutIndexTemplateRequest(templateName).source(template);
+    template.put("aliases", Collections.singletonMap(aliasName, Collections.emptyMap()));
 
-    return putIndexTemplate(request);
+    return putIndexTemplate(templateName, template);
   }
 
   /** @return true if request was acknowledged */
-  private boolean putIndexTemplate(final PutIndexTemplateRequest putIndexTemplateRequest) {
+  private boolean putIndexTemplate(final String templateName, final Object body) {
     try {
-      return client
-          .indices()
-          .putTemplate(putIndexTemplateRequest, RequestOptions.DEFAULT)
-          .isAcknowledged();
+      final var request = new Request("PUT", "/_template/" + templateName);
+      request.addParameter("include_type_name", "true");
+      request.setJsonEntity(MAPPER.writeValueAsString(body));
```

Comment:
Perhaps it cannot occur here, but on line 110 you catch the potentially thrown IOException. It feels like the same thing might happen here.

Reply:
Oh wait, I'm mistaken, the catch for this exception occurs already on line 205. Nevermind